tasks/translation/translate_content.py

```
@@ -0,0 +1,216 @@
+            provider = NoOpProvider()
+
+        enhanced: List[DocumentChunk] = []
+
+        for chunk in data_chunks:
+            text = getattr(chunk, "text", "") or ""
+            content_id = getattr(chunk, "id", getattr(chunk, "chunk_index", ""))
+            # detect language
+            try:
+                lang, conf = await provider.detect_language(text)
+            except Exception as e:
+                logger.exception("language detection failed: %s", e)
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)